CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+* Declarative and Reactive (Operator pattern) approach
+
+This short page shall help to get started with the projects and topics
+surrounding them.  If you notice a strong similarity with the [KubeVirt contribution guidelines](https://github.com/kubevirt/kubevirt/blob/main/CONTRIBUTING.md) it's because we have taken inspiration from their success.
+
+
+## Contributing to hppo
+
+### Our workflow
+
+Contributing to hppo should be as simple as possible. Have a question? Want
+to discuss something? Want to contribute something? Just open an
+[Issue](https://github.com/kubevirt/hostpath-provisioner-operator/issues) or a [Pull
+Request](https://github.com/kubevirt/hostpath-provisioner-operator/pulls).  For discussion, we use the [KubeVirt Google Group](https://groups.google.com/forum/#!forum/kubevirt-dev).
+
+If you spot a bug or want to change something pretty simple, just go
+ahead and open an Issue and/or a Pull Request, including your changes
+at [kubevirt/hostpath-provisioner-operator](https://github.com/kubevirt/hostpath-provisioner-operator).
+
+For bigger changes, please create a tracker Issue, describing what you want to
+do. Then either as the first commit in a Pull Request, or as an independent
+Pull Request, provide an **informal** design proposal of your intended changes.
+The location for such propoals is
+[/docs](docs/) in the hppo repository. Make sure that all your Pull Requests link back to the
+relevant Issues.
+
+### Getting started
+
+To make yourself comfortable with the code, you might want to work on some
+Issues marked with one or more of the following labels
+[help wanted](https://github.com/kubevirt/hostpath-provisioner-operator/labels/help%20wanted),
+[good first issue](https://github.com/kubevirt/hostpath-provisioner-operator/labels/good%20first%20issue),
+or [bug](https://github.com/kubevirt/hostpath-provisioner-operator/labels/kind%2Fbug).
+Any help is greatly appreciated.
+
+### Testing
+
+**Untested features do not exist**. To ensure that what we code really works,
+relevant flows should be covered via unit tests and functional tests. So when
+thinking about a contribution, also think about testability. All tests can be
+run local without the need of CI. Have a look at the
+[Developer Guide](hack/README.md).
+
+### Getting your code reviewed/merged
+
+Maintainers are here to help you enabling your use-case in a reasonable amount
+of time. The maintainers will try to review your code and give you productive
+feedback in a reasonable amount of time. However, if you are blocked on a
+review, or your Pull Request does not get the attention you think it deserves,
+reach out for us via Comments in your Issues, or ping us on
+[Slack](https://kubernetes.slack.com/messages/kubevirt-dev).
+
+Maintainers are:
+
+* @awels
+
+### PR Checklist
+
+Before your PR can be merged it must meet the following criteria:
+* [README.md](README.md) has been updated if core functionality is affected.
```

Comment:
here is how it looks:
`https://github.com/kubevirt/hostpath-provisioner-operator/blob/add_contributing_doc/README.md`